ui/shared/components/panel/variants/VariantIndividuals.jsx

```diff
@@ -152,7 +152,13 @@ export const Alleles = React.memo(({ genotype, variant, isHemiX, warning }) => (
         content={
           <div>
             <b>Warning: </b>
-            {warning}
+            {warning.split('\n').reduce((acc, message) => (acc ? (
```

Comment:
This is pretty confusing behavior. It looks like you are trying to get the first warning message to display inline with the "Warning:" header and then everything else to display on its own row. I think it would be better to pass in an array and check the type here, and then if its an array show each entry on its own row